system/jlib/jtrace.cpp

```diff
@@ -655,13 +886,22 @@ class CServerSpan : public CSpan
         return success;
     }
 
+    void setContextAttributes()
```

Comment:
I realized the Otel span object was not being annotated w/ our "passthrough" header values, we'll likely need this anyway